infrastructure/evault-provisioner/src/templates/evault.nomad.ts

```
@@ -37,9 +37,10 @@ export function generatePassword(length = 16): string {
  * @throws {Error} If the service endpoint cannot be determined from the cluster.
  */
 export async function provisionEVault(w3id: string, eVaultId: string) {
-    const idParts = w3id.split('@');
-    w3id = idParts[idParts.length - 1]
-    const neo4jPassword = sha256(w3id)
+    console.log("starting to provision");
+    const idParts = w3id.split("@");
+    w3id = idParts[idParts.length - 1];
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)